data/transactions/logic/eval.go

```diff
@@ -3081,6 +3118,40 @@ var ecdsaVerifyCosts = []int{
 	Secp256r1: 2500,
 }
 
+// ecdsaVerify cheks a signature,
```

Comment:
@jannotti Sanity checking:  Is `ecdsaVerify` modifications deliberately included in the PR?

We previously discussed these changes in https://github.com/algorand/go-algorand/pull/4368.  I'm trying to gauge if there's a mistaken merge and/or if you'd like to move discussion here.

Reply:
Yes, I just noticed that too.  I did it a while back when I assumed the panic check would go in soon, and I wanted to work on this in Go 1.19, and have passing tests.  I'll remove before merging.

Reply:
rebased away, like a ghost